[core]/es_extended/server/classes/player.lua

```diff
@@ -470,13 +470,17 @@ function CreateExtendedPlayer(playerId, identifier, group, accounts, inventory,
 					self.removeWeaponComponent(weaponName, v2)
 				end
 
+				local weaponHash = joaat(weaponName)
+				
+				RemoveWeaponFromPed(GetPlayerPed(self.source), weaponHash)
```

Comment:
GetPlayerPed delete please and declare the variable outside the for loop. I think 464 line and use this variable in the for loop.

Reply:
Moved all variables outside of the loop.

Reply:
I mean GetPlayerPed(self.source) not weaponHash.